working/augmentation-libraries/feature-specification.md

```diff
@@ -210,7 +210,7 @@ import augment 'b.dart';
 augment class C {}
 
 augment void trace() {
-  augment super.trace();
+  augmented.trace();
```

Comment:
Consider `augment.super.something`.

It would match what I'm considering for mixin constructors and composite mixins, where I'd do
```dart
class Foo extends Bar with Baz, Qux {
  Foo(Baz.super.arg, int argq)
      : Qux.super.named(argq), Foo.super();

  int get foo => Baz.super.foo + Foo.super.foo;
}
```

That is, in a scope with multiple "super"s, you qualify the `super` with which one it is, based on the type name which introduces it.
An `augment` declaration's augmented super would then be `augment.super`, which is not currently valid syntax, doesn't introduce a new word on top of `augment`, which I assume will be a built-in identifier, and therefore cannot conflict with a type name that one might otherwise want to do `X.super` on 

```dart
augment class Foo with MoreMixin {
  int get foo => augment.super.foo + MoreMixin.super.foo;
}
```